pkg/controller.v1/mpi/mpijob_controller.go

```diff
@@ -133,7 +133,8 @@ func (jc *MPIJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	}
 
 	if err = kubeflowv1.ValidateV1MpiJobSpec(&mpijob.Spec); err != nil {
-		logger.Info(err.Error(), "MPIJob failed validation", req.NamespacedName.String())
+		logger.Error(err, "MPIJob failed validation")
```

Comment:
Nice to add req.NamespacedName.String() as well in the log

Reply:
I think add req.NamespaceName.String() is redundant and strange as a value of KV.
As following shows, the first is from ```logger.Info(err.Error(), "MPIJob failed validation", req.NamespacedName.String())```, the second is from ```logger.Error(err, "MPIJob failed validation")```

```
1.6717681860476494e+09  INFO    PyTorchReplicaType is Master2 but must be one of [Master Worker]        {"pytorchjob": "default/pytorch-test-validate", "PyTorchJob failed validation": "default/pytorch-test-validate"}
1.6717681860476797e+09  ERROR   PyTorchJob failed validation    {"pytorchjob": "default/pytorch-test-validate", "error": "PyTorchReplicaType is Master2 but must be one of [Master Worker]"}
```

Reply:
SGTM